src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        Colors[] colors = Colors.values();
+        return colors[new Random().nextInt(colors.length)].toString();
```

Comment:
lets make Random a global variable, so it is going to be created just once instead of each time this method is invoked